build_system/src/build.rs

```diff
@@ -43,6 +43,7 @@ impl BuildArg {
         Ok(Some(build_arg))
     }
 
+    /// Displays usage information for the build command.
```

Comment:
Please remove this comment as it is a duplicate of the code itself, so it doesn't bring anything more than the code.